### ISSUE TITLE
Logging page edits

### ISSUE THREAD

**Issue by issmirnov** [cosmocode/dokuwiki-plugin-logstats#17]
Hello,

I wanted to ask if you would be open to a PR that implements logging of page edits (Action Mode: `edit`), in addition to page accesses (Action Mode: `show`). 

My use case and rationale:

- I'm trying to build a Kibana dashboard that will track edits by users. Since I'm interested specifically in the `edit` action, the `access.log` is not very helpful - I can attempt to create some convoluted grok rule that will check that the ref page has a `?do=edit` suffix but it's pretty painful. In addition, if I were to use the grep trick for referring page, then I would trigger accesses to media such as CSS, the logo, and other payloads.
- I can attempt to parse my nginx log directly, but I run into a similar issue with duplication of referral headers.
- Therefore, it would be simplest and least painful to inject this functionality into this extremely useful logging plugin, since then I can hook into the various dokuwiki util functions, the admin UI config page, and the already tested logging mechanism. I can then pull this edit log and feed it into my scripts.

My proposed approach:

- refactor `action.php` to contain methods `logShowAction` and `logEditAction` instead of `logAccess`. These will write to `access.log` and `edit.log` respectively.
- call one of these two functions in the tail of `logPageAccess`, branching on `$ACT`

I would love any feedback or suggestions you may wish to deliver, and if all is well I'd be happy to implement this and send in a PR within a few days. I think with the rise of various monitoring solutions it would benefit people to have this convenient file that they can pull.

**Comment by micgro42** [comment]
Wouldn't it be easier to use the  [authorstats](https://www.dokuwiki.org/plugin:authorstats)-plugin?

**Comment by issmirnov** [comment]
No, this solves a different problem. There are various frameworks such as the [ELK Stack](https://www.elastic.co/products) and other log aggregators that have plugins well suited for sucking up Apache/Nginx formatted logs from text files. 

The plugin you linked will only display that information on the wiki level, and will not write to a file.
